exp-timing/femto.py

```diff
@@ -58,124 +57,121 @@ def __init__(self, nx='NULL'):
 
         nm = 'XCS'
         namelist.add(nm)
-        base = 'LAS:FS4:'  # base name for this sysetm
-        dev_base[nm] = base+'VIT:'
+        base = 'LAS:FS4:'  # Base PV name for this laser locker.
+        dev_base[nm] = base+'VIT:' 
         matlab_pv_base[nm] = dev_base[nm]+'matlab:'
-        matlab_pv_offset[nm] = 1
+        matlab_pv_offset[nm] = 1 # Notepad PV numbering starts at 1.
         matlab_pv_digits[nm] = 2
-        counter_base[nm] = base+'CNT:TI:'   # time interval counter
+        counter_base[nm] = base+'CNT:TI:'   # PV name for the Time Interval Counter (SR620)
         freq_counter[nm] = dev_base[nm]+'FREQ_CUR'        
         phase_motor[nm] = base+'MMS:PH' 
         error_pv_name[nm] = dev_base[nm]+'FS_STATUS' 
         version_pv_name[nm] = dev_base[nm]+'FS_WATCHDOG.DESC' 
-        laser_trigger[nm] = 'EVR:LAS:XCS:01:TRIG0:TDES' # was 'LAS:SR63:EVR:09:CTRL.DG0D'
-        trig_in_ticks[nm] = 0  # experiment side triggers operate in ticks units
-        reverse_counter[nm] = 1
-        use_secondary_calibration[nm] = 0
+        laser_trigger[nm] = 'EVR:LAS:XCS:01:TRIG0:TDES' # Name of on time EVR trigger channel.
+        trig_in_ticks[nm] = 0  # The laser_trigger PV is the delay value in ns. 0 sets trigger value to ns.
+        reverse_counter[nm] = 1 # A value of 1 sets the laser as the counter start and an EVR trigger the counter stop.
+        use_secondary_calibration[nm] = 0 # Secondary calibration turned off. 
         matlab_use = dict()
         for n in range(0,20):
-            matlab_use[n] = False  # Use new PVs
-        # modified for timetool drift draft
-        drift_correction_signal[nm] = 'LAS:FS4:VIT:matlab:29' # what PV to read
-        drift_correction_multiplier[nm] = -1/(2.856 * 360); 
-        drift_correction_value[nm]= 'LAS:FS4:VIT:matlab:04'# PV the current reading in ns.
-        drift_correction_offset[nm]= 'LAS:FS4:VIT:matlab:05' # PV in final nanoseconds
-        drift_correction_gain[nm]= 'LAS:FS4:VIT:matlab:06'  # PV nanoseconds / pv value, 0 is disable
-        drift_correction_dir[nm]= 1  # Direction of timetool stage
-        drift_correction_smoothing[nm]='LAS:FS4:VIT:matlab:07'
-        drift_correction_accum[nm]='LAS:FS4:VIT:matlab:09'
+            matlab_use[n] = False  # Use IOC PVs.
+        # Notepad PVs for drift correction feature.
+        drift_correction_signal[nm] = 'LAS:FS4:VIT:matlab:29' # Drift correction value that is pulled from the time_tool.py script.
+        drift_correction_value[nm] = 'LAS:FS4:VIT:matlab:04' # PV of the current drift correction value in ns.
+        drift_correction_offset[nm] = 'LAS:FS4:VIT:matlab:05' # Allows a fixed offset to be applied to the 'fs' value from the timetool script.
+        drift_correction_gain[nm] = 'LAS:FS4:VIT:matlab:06'  # 0 is disable.
```

Comment:
And what about if it's not 0?

Reply:
Added clarification drift_correction_gain comment in the initial dict declarations.